norihiro/obs-color-monitor
# Update translation and add Japanese translation

### Description

Add missing translations to en-US.ini
I noticed a lot of translation texts are missing when reviewing #69.

### How Has This Been Tested?


Check texts are translated during these steps.
1. Add a scope dock.
2. Open the right-click menu.
3. Open Properties and check all tabs.


### General checklist
- [ ] The commit is reviewed by yourself.
- [x] The code is tested.
- [x] Document is up to date or not necessary to be changed.
- [x] The commit is compatible with repository's license.
